library/Notifications/Widget/ItemList/ContactList.php

```diff
@@ -14,4 +15,10 @@ protected function getItemClass(): string
     {
         return ContactListItem::class;
     }
+
+    protected function init(): void
+    {
+        $this->getAttributes()
+            ->set('data-icinga-detail-url', (string) Url::fromPath('notifications/contact'));
```

Comment:
Same change as for `ContactGroupList`.
Remember to introduce null values to `Links::contact()` as well.